velox/core/PlanNode.h

```diff
@@ -2957,7 +2957,8 @@ class HashJoinNode : public AbstractJoinNode {
       TypedExprPtr filter,
       PlanNodePtr left,
       PlanNodePtr right,
-      RowTypePtr outputType)
+      RowTypePtr outputType,
+      void* reusedHashTableAddress = nullptr)
```

Comment:
@xiaoxmeng it adds one more parameter(with default value) for `HashJoinNode`, instead of modifying the existing API, is it better to add a new set of one (like the `IndexLookupJoinNode`)?